pkgs/by-name/va/vapoursynth/package.nix

```diff
@@ -17,13 +17,13 @@
 
 stdenv.mkDerivation rec {
   pname = "vapoursynth";
-  version = "72";
+  version = "71";
```

Comment:
> [!NOTE]
> This change is done because the `vapoursynth` update broke `python3Packages.vapoursynth`
- see: https://github.com/NixOS/nixpkgs/pull/413321